src/windows/window.ts

```diff
@@ -159,6 +161,24 @@ export default () => {
   }
 
   api.setWindowInfo = info => {
+    if (info.is_float) {
+      const { x, y } = api.positionToWorkspacePixels(info.row, info.col)
+      const xPx = `${x}px`
+      const yPx = `${y}px`
+
+      Object.assign(nameplate.element.style, {
```

Comment:
Maybe just remove the nameplate on floating windows completely? It seems like having it around in the DOM is less efficient than it could be since it isn't even used/displayed.